marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { fireEvent, render } from "@testing-library/react";
+import QueryExportModal from './query-export-modal';
+
+
+describe('Query Export Component', () => {
+
+  let columns = ['id', 'firstName', 'lastName', 'age']
+
+  test('Verify Query Export Modal Dialog renders', () => {
+    const { getByTestId, getByText } = render(<QueryExportModal exportModalVisibility={true} columns={columns} />);
+    expect(getByTestId('query-export-form')).toBeInTheDocument();
+    expect(getByText('Rows:')).toBeInTheDocument();
+    expect(getByText('All')).toBeInTheDocument();
+    expect(getByText('Limited set of the first rows returned')).toBeInTheDocument();
+    expect(getByText('Export to a CSV file containing the columns of data currently displayed.')).toBeInTheDocument();
+  });
+
+  test('Verify able to select All rows', () => {
+    const { getByText } = render(<QueryExportModal exportModalVisibility={true} columns={columns} />);
+    fireEvent.click(getByText('All'));
+  });
+
+  test('Verify able to select Limited number of rows', () => {
+    const { getByText } = render(<QueryExportModal exportModalVisibility={true} columns={columns} />);
+    fireEvent.click(getByText('Limited set of the first rows returned'));
```

Comment:
We can remove this test, since it is being verified in line 38 and it is better there since there are assertions following the click event. This test with 1 line of click can be deleted as it adds no value.

Reply:
Thanks Srikanth, I added fix in the pr 3937.